test/e2e/run_test.go

```diff
@@ -1212,14 +1212,14 @@ USER mail`, BB)
 	})
 
 	It("podman run with bad healthcheck timeout", func() {
-		session := podmanTest.Podman([]string{"run", "-dt", "--health-cmd", "[\"foo\"]", "--health-timeout", "0s", ALPINE, "top"})
+		session := podmanTest.Podman([]string{"run", "-dt", "--health-cmd", "foo", "--health-timeout", "0s", ALPINE, "top"})
```

Comment:
Why change this? Is the syntax no longer legal?

Reply:
No it is, I just thought It would be good to alternate the tests with strings and arrays to test both